src/scss/_forms.scss

```diff
@@ -261,6 +261,16 @@
     }
   }
 
+  &.-two-col-secrets {
+    flex-flow: row wrap;
+    gap: 0.5rem;
+    max-width: 30%;
```

Comment:
i'll check on this, but not a fan to create a one-off class here. that means the original is not flexible enough to handle this scenario.

Reply:
i'll open separate PR